src/tokenManagement/tokenManagementController.ts

```diff
@@ -0,0 +1,495 @@
+import { IAccount } from "../accounts/interfaces";
+import { TransactionOnNetwork } from "../networkProviders";
+import { INetworkProvider } from "../networkProviders/interface";
+import { IESDTIssueOutcome } from "../tokenOperations";
```

Comment:
the `tokenOperations` package will be removed and we should not import from there.

Reply:
I will to this when I will remove the package